src/javascripts/jquery.tipsy.js

```diff
@@ -118,6 +118,21 @@
                 }
             }
         },
+
+        destroy: function(){
+          $.removeData(this.$element.get(0));
```

Comment:
`$.removeData()` is a lower-level API and in the example they use the raw DOM element, too. See: https://api.jquery.com/jQuery.removeData/.